content/docs/user-guide/project-structure/pipelines-files.md

```diff
@@ -114,6 +114,35 @@ stages:
 The commands in `dvc metrics` and `dvc plots` help you display and compare
 metrics and plots.
 
+## Plots definitions
+
+The list of plots contains one or more user-defined
+[plots](/doc/command-reference/plots#plots-definitions). Here's an example that
+tells DVC that `auc.json` is viable for visualization:
+
+```yaml
+stages:
+  build:
+    cmd: python train.py
+    deps:
+      - features.csv
+    outs:
+      - model.pt
+      - auc.json
+    metrics:
+      - accuracy.txt:
+          cache: false
+plots:
+  auc.json:
+    x: fpr
+    y: tpr
+```
+
+Note that in this example we don't have to specify `auc.json` as a plot output.
```

Comment:
It might help to explain why this is useful and why there are two different ways to define plots. Maybe provide an example that uses multiple files or otherwise illustrates its use?